client/my-sites/plans-single-products/index.jsx

```diff
@@ -92,11 +101,15 @@ export default connect(
 					title: 'Jetpack Backup',
 				},
 			},
+			selectedProduct: 'jetpack_backup_realtime',
 		};
 	},
 	() => {
 		return {
-			onProductSelect: () => {
+			onProductChange: () => {
```

Comment:
I'm not sure this will need to be in the URL, this it might be better to be implemented as a component state.